crates/ruff/src/rules/pyflakes/fixes.rs

```diff
@@ -90,31 +90,28 @@ pub(crate) fn remove_unused_positional_arguments_from_format_call(
 
 /// Generate a [`Edit`] to remove the binding from an exception handler.
 pub(crate) fn remove_exception_handler_assignment(
-    except_handler: &ExceptHandler,
+    range: TextRange,
```

Comment:
Nit: Can we give this range a more specific name: E.g. `except_handler_range`? I think it's also fine to pass in the except handler even if we only use its range. It acts as "parameter documentation"